pom.xml

```diff
@@ -110,10 +108,17 @@
             <version>2.2.0</version>
         </dependency>
 
+        <!-- From jenkins 2.60.3 -->
```

Comment:
Not sure this is the best way. If you've got a conflict somewhere and this is pulled in by the core, a good way is to exclude it from other dependencies. This will allow the plugin to always behave with the version coming from the Core.

Doing this enforced here is going to cause differences when the Core is bumped and you miss aligning it to a possibly updated version in the Core dependencies.

Reply:
Can you give me an example here?  In this case I'm trying to use the existing conventions to reduce the changes.  It sounds like you are offering an improvement on the existing codebase?  I think another reviewer earlier commented on something as was set to `provided`, and it may not have actually been provided by core.  I think `javax.servlet-api` might have been one example where I had to add the dependency to the POM for it to compile on java8 (vs just the tests compiling).  In that case you are just recommending remove the version tag from the dependency?

Reply:
I think I figured it out, this should be resolved